src/api/controllers/cache-controller.ts

```diff
@@ -3,6 +3,7 @@ import * as prom from 'prom-client';
 import { bufferToHexPrefixString, logger, normalizeHashString } from '../../helpers';
 import { asyncHandler } from '../async-handler';
 import { PgStore } from '../../datastore/pg-store';
+import { sha256 } from 'bitcoinjs-lib/src/crypto';
```

Comment:
Bitcoinjs-lib uses a slow js implementation of sha256. Can you use the built-in nodejs one?
```js
import { createHash } from 'node:crypto'

function sha256(content) {  
  return createHash('sha256').update(content).digest('hex')
}
```